app/layout.tsx

```diff
@@ -0,0 +1,54 @@
+import './globals.css';
+import { Source_Sans_3, Source_Serif_4 } from 'next/font/google';
+import { storyblokInit, apiPlugin } from '@storyblok/react/rsc';
+import StoryblokProvider from '@/components/StoryblokProvider';
+import { FlexBox } from '@/components/FlexBox';
+import { Masthead } from '@/components/Masthead';
+import { LocalFooter } from '@/components/LocalFooter';
+import { GlobalFooter } from '@/components/GlobalFooter';
+import { Skiplink } from '@/components/SkipLink';
+
+type LayoutProps = {
+  children: React.ReactNode,
+};
+
+const sourceSans = Source_Sans_3({
+  subsets: ['latin'],
+  display: 'swap',
+  variable: '--font-source-sans',
+});
+
+const sourceSerif = Source_Serif_4({
+  subsets: ['latin'],
+  display: 'swap',
+  variable: '--font-source-serif',
+});
+
+storyblokInit({
+  accessToken: process.env.STORYBLOK_ACCESS_TOKEN,
+  use: [apiPlugin],
+  apiOptions: {
+    region: 'us',
+  },
+});
+
+export default function RootLayout({ children }: LayoutProps) {
+  return (
+    <StoryblokProvider>
+      <html lang="en" className={`${sourceSans.variable} ${sourceSerif.variable}`}>
```

Comment:
All necessary html attributes and font related classes can be added directly here in `layout.tsx`
https://nextjs.org/docs/app/building-your-application/routing/pages-and-layouts#layouts